bylaws/009-per-workflow.md

```diff
@@ -0,0 +1,85 @@
+# PER Workflow
+
+A PHP Evolving Recommendation (PER) is a formal definition of best practices, utilities, reference implementations, and similar tools.  They may evolve over time as the PHP ecosystem develops.  
+
+## Pre-Draft
+
+The goal of the Pre-Draft stage is to determine whether a majority of the PHP FIG is interested in establishing a PER Working Group for a proposed concept.
+
+Interested parties may discuss a possible proposal, including possible implementations, by whatever means they feel is appropriate. That includes informal discussion on official FIG discussion mediums of whether or not the idea has merit and is within the scope of the PHP FIG's goals.
+
+Once those parties have determined to move forward, they must form a Working Group. A Working Group consists of, at minimum:
+
+* One Editor
+* One Core Committee member who will act as Sponsor
+* At least three other individuals. These may include Project Representatives, Secretaries or Core Committee members as well as members of the general community.
+
+The proposal is not required to be fully developed at this point, although that is permitted. At minimum, it must include a statement of the problem to be solved and basic information on the general approach to be taken. Further revision and expansion is expected during the Draft Phase.
+
+The Sponsor may then call for an Entrance Vote of the Core Committee to enquire whether the Core Committee is generally interested in maintaining a PER for the proposed subject, even if they disagree with the details of the proposal.
+
+If the vote passes, the proposal officially enters Draft stage. The proposal is given a unique descriptive name (such as "CodingStandards", "CacheUtil", etc.).
```

Comment:
Naming convention: is there a reason for the descriptive names to be in camel case?
I thought that these names were meant for humans, allowing spaces to be used (ex `Coding Standards`). Defining the package name, kebab-case convention should be used.

Reply:
Not in particular, beyond knee-jerk preference for strings that can be identifiers.  In this case it probably is better to let it be longer.